packages/cli/src/deploy/deploy.ts

```diff
@@ -62,9 +61,29 @@ export async function deploy({
 }: DeployOptions): Promise<WorldDeploy> {
   const deployerAddress = initialDeployerAddress ?? (await ensureDeployer(client));
 
-  await ensureWorldFactory(client, deployerAddress, config.deploy.upgradeableWorldImplementation);
```

Comment:
this happens inside `deployWorld`, so no need to double up on it here (or do it when a custom world is used)